src/applications/simple-forms/20-10206/sass/10206-pa.scss

```diff
@@ -6,6 +6,13 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../../platform/forms/sass/m-form-confirmation";
 
+// TODO: Remove this rule below once SaveInProgressIntro's
+// hide-unauth'ed-content support's fixed/refactored (#914)
+.usa-alert-info.schemaform-sip-alert .usa-alert-text > p:nth-child(2) {
```

Comment:
I'd prefer to leave the bug visible to prove that the refactor if needed, and just do the refactor.  is this 
bandaid urgent?

Reply:
Yes, we're launching this afternoon.